draft-kowalik-rpp-architecture-01.adoc

```diff
@@ -367,6 +367,18 @@ Operations shall be defined in IANA registry in a way that allows for automated
 FIXME: find an appropriate section for this
 *  Compatibility Profiles - to define subsets of RPP for specific use cases or EPP compatibility.
 
+==== Extension mechanisms

```

Comment:
I like this chapter and I think it shall be on the main level within "Protocol Details" because it addresses  a cross-layer concern, not only "Resource Definition Layer".